libkbfs/tlf_edit_history_test.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol"
+	"golang.org/x/net/context"
+)
+
+func TestBasicTlfEditHistory(t *testing.T) {
+	var userName1, userName2 libkb.NormalizedUsername = "u1", "u2"
+	config1, _, ctx := kbfsOpsConcurInit(t, userName1, userName2)
+	defer CheckConfigAndShutdown(t, config1)
+
+	clock, now := newTestClockAndTimeNow()
+	config1.SetClock(clock)
+
+	config2 := ConfigAsUser(config1.(*ConfigLocal), userName2)
+	defer CheckConfigAndShutdown(t, config2)
+
+	name := userName1.String() + "," + userName2.String()
+
+	rootNode1 := GetRootNodeOrBust(t, config1, name, false)
+	rootNode2 := GetRootNodeOrBust(t, config2, name, false)
+
+	// user 1 creates a file
+	kbfsOps1 := config1.KBFSOps()
+	_, _, err := kbfsOps1.CreateFile(ctx, rootNode1, "a", false, NoExcl)
+	if err != nil {
```

Comment:
Since this is a new test file, have you considered using `require`? Would cut down on a lot of the `if err != nil` lines, and usually the error messages are good enough to not require custom messages.